docs/source/admin/quick_howto/static_dns.rst

```diff
@@ -0,0 +1,77 @@
+..
+..
+.. Licensed under the Apache License, Version 2.0 (the "License");
+.. you may not use this file except in compliance with the License.
+.. You may obtain a copy of the License at
+..
+..     http://www.apache.org/licenses/LICENSE-2.0
+..
+.. Unless required by applicable law or agreed to in writing, software
+.. distributed under the License is distributed on an "AS IS" BASIS,
+.. WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+.. See the License for the specific language governing permissions and
+.. limitations under the License.
+..
+
+.. _static-dns-qht:
+
+******************************
+Configuring Static DNS Entries
+******************************
+Any number static DNS entries can be configured on a per-:term:`Delivery Service` basis. In a typical scenario, the :term:`Delivery Service` will have DNS entries automatically generated based on its "xml_id" and "routing name", and the name and subdomain of the CDN to which it belongs. For example, in the :ref:`ciab` default environment, the "demo1" :term:`Delivery Service` has an automatically generated DNS record for ``video.demo1.mycdn.ciab.test``. Configuring a static DNS entry allows for further extension of this, for example, one could create an entry that enforces lookups of the name ``foo.video.demo1.mycdn.ciab.test`` resolve to the IPv4 address ``0.0.0.1``.
```

Comment:
Example IP addresses should probably be 10.* or 192.168.*

`foo.video.demo1.mycdn.ciab.test` would probably make more sense as `foo.demo1.mycdn.ciab.test` so that it doesn't look like `video` is a subdomain (since `video` is actually the routing name).

"one could create an entry" -> "one could create an A record"

Reply:
> _"Example IP addresses should probably be 10.* or 192.168.*"_

Why?

> _"foo.video.demo1.mycdn.ciab.test would probably make more sense as foo.demo1.mycdn.ciab.test so that it doesn't look like video is a subdomain (since video is actually the routing name)."_

seems I misunderstood how that works...

> _"'one could create an entry' -> 'one could create an A record'"_

Then, in general "entry" -> "record"? Semantically, what's the difference between a DNS "entry" and a DNS "record"? Does the former ever exist or is it just something we're making up?

Reply:
Example IPs should be `192.0.2.*` and `2001:DB8::*`. 

Because 
https://tools.ietf.org/html/rfc5737 
https://tools.ietf.org/html/rfc3849

Reply:
> _"This document is not an Internet Standards Track specification; it is
   published for informational purposes."_

Not that it matters to me what the example IP is, but this doesn't seem to be a real standard. And where does that leave `10.*`?

Reply:
>this doesn't seem to be a real standard

Many RFCs are not Internet Standards Track. These RFCs are authoritative, they reserve these IP subnets, and they are not to be used or assigned on the internet.

Moreover, the IANA is authoritative:
https://www.iana.org/assignments/iana-ipv4-special-registry/iana-ipv4-special-registry.xhtml
https://www.iana.org/assignments/iana-ipv6-special-registry/iana-ipv6-special-registry.xhtml

> where does that leave 10.*?

`10.*` is reserved for internal subnets. See https://tools.ietf.org/html/rfc1918. We shouldn't use it for documentation, RFC5737 and 3849 authoritatively specify the correct subnets for docs.

Reply:
>Semantically, what's the difference between a DNS "entry" and a DNS "record"? Does the former ever exist or is it just something we're making up?

"Record" is the canonical term. Older DNS RFCs occasionally use the term "entry," and other people do use it, not just us. But "record" is the canonically correct term.

See:
https://tools.ietf.org/html/rfc1035#section-4.1.3
https://www.iana.org/assignments/dns-parameters/dns-parameters.xhtml#dns-parameters-4
https://en.wikipedia.org/wiki/List_of_DNS_record_types

Reply:
Neat. I'll leave the page name, menu option name, and docs section name alone so that they all match the API route, but I'll prefer "record" to "entry" elsewhere.

Reply:
> 10.* is reserved for internal subnets. See https://tools.ietf.org/html/rfc1918. We shouldn't use it for documentation, RFC5737 and 3849 authoritatively specify the correct subnets for docs.

Yeah, that's right. I was thinking rfc1918 would be better than 0.0.0.1 but was also too lazy to look up the rfc for the recommended documentation CIDRs. Thanks for doing that!